packages/hooks/jest.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
this is fine for now, but the approach in https://github.com/Developer-DAO/web3-ui/pull/25 would delete it in favor of the root jest config